content/en/docs/tutorials/security/seccomp.md

```diff
@@ -482,15 +482,15 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
   - role: control-plane
-    image: kindest/node:v1.23.0@sha256:49824ab1727c04e56a21a5d8372a402fcd32ea51ac96a2706a12af38934f81ac
+    image: kindest/node:v1.28.0
```

Comment:
we use to publish the sha for kind images

Reply:
> we use to publish the sha for kind images

Adjusted @aojea 

[DockerHubReferenceLink](https://hub.docker.com/layers/kindest/node/v1.28.0/images/sha256-9f3ff58f19dcf1a0611d11e8ac989fdb30a28f40f236f59f0bea31fb956ccf5c?context=explore)